freedomOfComputing/src/main/java/nl/tudelft/trustchain/foc/community/FOCVote.kt

```diff
@@ -2,4 +2,4 @@ package nl.tudelft.trustchain.foc.community
 
 import java.io.Serializable
 
-data class FOCVote(val memberId: String) : Serializable
```

Comment:
I would have it be an enum, because here there is nothing that prevents us from passing -1. So just make a 
`enum class VoteType {
    UP, DOWN
}`
and have FOCVote take that instead of an Int